### PR TITLE
OpInfos for: normal, bernoulli, multinomial

### PR DESCRIPTION
**Pull request by zou3519**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#66358 OpInfos for: normal, bernoulli, multinomial**

Test Plan:
- run tests

Differential Revision: [D31551695](https://our.internmc.facebook.com/intern/diff/D31551695)

### CODE REVIEW

**Review by mruberry — APPROVED** [review]
Cool! I left a few minor inline comments. I'm not sure what's up with the lint build failure, however.

**Comment by Lezcano** [comment]
Is there anything blocking this PR? It'd be nice to have it landed to be able to test the stack in https://github.com/pytorch/pytorch/pull/69631

**Comment by zou3519** [comment]
@Lezcano I can rebase and land this

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -2075,6 +2075,73 @@ def get_val(dtype):
...
+
+    return sample_inputs_normal_common(self, device, dtype, requires_grad, cases, **kwargs)
+
+def sample_inputs_normal_tensor_second(self, device, dtype, requires_grad, **kwargs):
+    cases = [
+        ([3, 4], 0.3, {}),
+    ]
+    return sample_inputs_normal_common(self, device, dtype, requires_grad, cases, **kwargs)
+
+def sample_inputs_bernoulli(self, device, dtype, requires_grad, **kwargs):
```
[the remark this answers is not in the document]
Comment (zou3519) [reply torch/testing/_internal/common_methods_invocations.py:2129]: Going to file an issue for generator support

**Comment by zou3519** [comment]
@zou3519 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31551695).

**Comment by zou3519** [comment]
@zou3519 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31551695).

**Pull request closed without merging**